typing/includemod.ml

```diff
@@ -682,17 +700,20 @@ and signature_components  ~in_eq ~loc old_env ~mark env subst
               | _ -> true
             in
             let shape_map = Shape.Map.add_value_proj shape_map id1 orig_shape in
+            let item = recoverable_error item in
```

Comment:
Reading the code, I thought that `item` was error information, and I was confused as I didn't know how you could tell from this code that `id1`, `id2` don't match or something. In fact `recoverable_error` means "if this item is an error, then mark that error as recoverable". I think `with_recoverable_error` or `mark_errors_recoverable` would be clearer.

```ocaml
let item = mark_errors_recoverable @@ 
  value_descriptions ...
```

Reply:
Let's go with `mark_error_as_recoverable`.